vue/src/components/editor/ToolManager.js

```diff
@@ -125,6 +180,89 @@ class DeleteTool extends Tool {
   }
 }
 
+class MoveTool extends Tool {
```

Comment:
This code doesn't seem to be working beside the tool icon showing up in the menu.